curated_transformers/layers/__init__.py

```diff
@@ -17,7 +18,37 @@
 from .normalization import RMSNorm
 from .transformer import (
     DecoderLayer,
+    EmbeddingsDropouts,
+    EmbeddingsLayerNorms,
```

Comment:
Wondering if `EmbeddingDropouts` and `EmbeddingLayerNorms` sound better.

Reply:
Maybe 🤔, should also be in a separate PR.